### PR TITLE
Update docs for the format of the new website

### PR DESCRIPTION
**Pull request by yangsoon**
as title

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      ],
+    },
+    footer: {
+      links: [
+        {
+          title: 'Documentation',
+          items: [
+            {
+              label: 'Getting Started',
+              to: 'https://kubevela.io/docs/getting-started/install',
```
Comment (wonderflow) [inline docs/docusaurus.config.js:66]: does the relative path work?

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    ],
+  ],
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          sidebarPath: require.resolve('./sidebars.js'),
+          editUrl:
+            'https://github.com/oam-dev/kubevela.io/edit/main/',
```
Comment (wonderflow) [inline docs/docusaurus.config.js:140]: what does this url for?

File: docs/docusaurus.config.js (hunk 1 above)
Comment (yangsoon) [reply docs/docusaurus.config.js]: ok

File: docs/en/helm/component.md [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 
 In this section, it will introduce how to declare Helm charts as app components via `ComponentDefinition`.
 
-> Before reading this part, please make sure you've learned [the definition and template concepts](../platform-engineers/definition-and-templates.md).
+> Before reading this part, please make sure you've learned [the definition and template concepts](../platform-engineers/definition-and-templates).
 
 ## Prerequisite
 
-* [fluxcd/flux2](../install.md#3-optional-install-flux2), make sure you have installed the flux2 in the [installation guide](https://kubevela.io/#/en/install).
+* [fluxcd/flux2](../install#3-optional-install-flux2), make sure you have installed the flux2 in the [installation guide](https://kubevela.io/#/en/install).
```
Comment (wonderflow) [inline docs/en/helm/component.md:11]: please change to relative link

File: docs/en/helm/component.md [hunk 4]
```diff
@@ -36,7 +38,7 @@ spec:
 ```
 
 In detail:
-- `.spec.workload` is required to indicate the workload type of this Helm based component. Please also check for [Known Limitations](/en/helm/known-issues?id=workload-type-indicator) if you have multiple workloads packaged in one chart.
+- `.spec.workload` is required to indicate the workload type of this Helm based component. Please also check for [Known Limitations](/docs/helm/known-issues?id=workload-type-indicator) if you have multiple workloads packaged in one chart.
```
Comment (wonderflow) [inline docs/en/helm/component.md:41]:
```suggestion
- `.spec.workload` is required to indicate the workload type of this Helm based component. Please also check for [Known Limitations](/docs/helm/known-issues#workload-type-indicator) if you have multiple workloads packaged in one chart.
```

File: docs/en/platform-engineers/keda.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
-# Defining KEDA as Autoscaling Trait
+---
+title:  Defining KEDA as Autoscaling Trait
+---
 
-> Before continue, make sure you have learned about the concepts of [Definition Objects](definition-and-templates.md) and [Defining Traits with CUE](https://kubevela.io/#/en/cue/trait) section.
+> Before continue, make sure you have learned about the concepts of [Definition Objects](definition-and-templates) and [Defining Traits with CUE](https://kubevela.io/#/en/cue/trait) section.
```
Comment (wonderflow) [inline docs/en/platform-engineers/keda.md:5]: relative docs link

File: docs/en/helm/component.md (hunk 3 above)
Comment (yangsoon) [reply docs/en/helm/component.md:11]: got it

File: docs/sidebars.js [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+            'developers/learn-appfile',
+          ]
+        },
+        {
+          'Command Line Tool (CLI)': [
+            'developers/port-forward',
+            'developers/check-logs',
+            'developers/exec-cmd',
+            'developers/cap-center',
+            'developers/config-enviroments',
```
Comment (wonderflow) [inline docs/sidebars.js:92]: please make the config environments to the first in CLI

**Review by wonderflow — APPROVED** [review]

**Pull request merged**